### ISSUE TITLE
Face recognition model in face_sdk

### ISSUE THREAD

**Issue by buiduchanh** [JDAI-CV/FaceX-Zoo#54]
Hi. @wang21jun . Thank for your contribute. 
I have a question about the model using for recognition, I saw many pretrain model in https://github.com/JDAI-CV/FaceX-Zoo/tree/main/training_mode but I dont know what model you used ? . In config file I saw this model name is face_recognition_1.0 but I dont know the architecture of this model ( backbone, head v..v). Can you share me the details of this model ? 
Thanks.

**Comment by wang21jun** [comment]
backbone: MobileFaceNet
head: MV-Softmax
accuracy on megaface: 90.39
you can get the model by this tool: https://github.com/JDAI-CV/FaceX-Zoo/blob/main/face_sdk/utils/model_convert.py

**Comment by buiduchanh** [comment]
> backbone: MobileFaceNet
> head: MV-Softmax
> accuracy on megaface: 90.39
> you can get the model by this tool: https://github.com/JDAI-CV/FaceX-Zoo/blob/main/face_sdk/utils/model_convert.py

Thanks for your quick reply.
By the way, I saw some sota models (backbone or head) and your explanation in this link https://github.com/JDAI-CV/FaceX-Zoo/issues/11#issuecomment-766672505. I wanna train a full model with strongest backbone and head ( Attention + MV-Softmax). I saw you used MS-Celeb and remove some identities so training set contain 72,778 identities and about 3.28M images. Can you share your preprocessed dataset ? 
Thanks you

**Comment by wang21jun** [comment]
https://github.com/JDAI-CV/FaceX-Zoo/blob/main/data/files/MS-Celeb-1M-v1c-r_id_list.txt
Just use all of the images of these identities.

**Comment by ooodragon94** [comment]
hi
thanks for your great repo!

I have trouble finding "https://github.com/JDAI-CV/FaceX-Zoo/blob/main/face_sdk/utils/model_convert.py"
can you help me out?

huge thanks!
